.rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+use_small_heuristics = "Max"
+
+# These settings are no-ops on stable.
+# To run them, use `cargo +nightly fmt`.
+imports_granularity = "Crate"
+group_imports = "StdExternalCrate"
```

Comment:
Do we want to require running the nightly rustfmt on an OpenSource project? This is quite annoying to set up for any external contributors.

Reply:
Good point. Let's not require it. But I'll still keep the formatting from `cargo +nightly fmt` run. For example, imports are now grouped.